packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ImageAndTextViewer/ImageAndTextViewer.stories.js

```diff
@@ -1,17 +1,16 @@
 import { Box, Grommet } from 'grommet'
 import { Provider } from 'mobx-react'
 import React from 'react'
-import asyncStates from '@zooniverse/async-states'
 import zooTheme from '@zooniverse/grommet-theme'
 
 import { SubjectFactory } from '@test/factories'
 import mockStore from '@test/mockStore'
 
-import ImageAndTextViewerConnector from './ImageAndTextViewerConnector'
+import SubjectViewer from '../../SubjectViewer'
 
 export default {
   title: 'Subject Viewers / ImageAndTextViewer',
-  component: ImageAndTextViewerConnector,
+  component: SubjectViewer,
```

Comment:
I’m a bit confused. Is this a story for `SubjectViewer`?

Reply:
Fixed